logging/OpenShift.md

```diff
@@ -46,22 +46,64 @@ oc login [cluster-hostname] -u [userID]
 ```
 
 ## <a name="l_os_cust"></a>Customization 
+#### General Customization
 
 Customization of SAS Viya Logging on OpenShift deployment follows the same process as in a standard logging deployment, which uses the USER_DIR environment variable to specify the location of your customization files. See the [logging README](../logging/README.md#log_custom) for information about the customization process.
 
-To change the number of days that log messages from OpenShift infrastructure components 
-are retained, modify the `INFRA_LOG_RETENTION_PERIOD` environment variable. The default 
-value is `1` (1 day).
+### OpenShift-Specific Customization
```

Comment:
Some of the info below isn't really about customization...it is just documentation on OpenShift "stuff" (e.g. the discussion of routes).

Reply:
Changed to OpenShift-Specific Customization and Information